src/tally.F90

```diff
@@ -441,6 +441,69 @@ subroutine score_general_ce(p, t, start_index, filter_index, i_nuclide, &
         end if
 
 
+      case (SCORE_PROMPT_NU_FISSION)
```

Comment:
I just pushed a modification such that there is only one routine, `score_fission_eout_ce`, which accepts the `score_bin` as an argument and scores the emitted neutrons if their emission (prompt or delayed) agrees with the requested emission type.